safe-global/safe-wallet-web
# Update the border in the dialogs and remove border for swap order history

## What it solves
Fix separators in different sections of interface

Resolves #3870

## How this PR fixes it
I have updated border from 2px to 1px for the dialogs like `create entry`, `edit entry`, `edit signer name`, `export address book` and `import address book`.

And removed the border from swap order on transaction history page.

## How to test it

Go the transaction history page and expand the swap order history and you see no border for it.
Go to the address book page click on the following buttons to see the dialog  `create entry`, `edit entry`,`export address book` and `import address book`.

## Screenshots
<img width="474" alt="Screenshot 2024-07-03 at 12 02 28 AM" src="https://github.com/safe-global/safe-wallet-web/assets/68734123/1bef81b4-d884-4467-8b9b-aeeb1e630d53">
<img width="475" alt="Screenshot 2024-07-03 at 12 02 41 AM" src="https://github.com/safe-global/safe-wallet-web/assets/68734123/0d25480b-9091-424b-a964-6e945872939c">


## Checklist
